### DISCUSSION TITLE
Undefined macros and classes

### DISCUSSION THREAD

**Discussion by faayez-10xe** [openhwfoundation/core-v-mcu-uvm#11] [Q&A]
Hi,
I am using this repo [https://github.com/openhwgroup/core-v-mcu-uvm] and compiling according to the README file after completing all  the pre-requisites mentioned in README. But while executing this command "mio sim uvmt_cvmcu_chip -t reg_bit_bash -s 1 -v high -w -a mdc" in Simulation Section I have faced the following errors: (screenshot is attached below)
![Screenshot from 2024-02-19 18-58-11](https://github.com/openhwgroup/core-v-mcu-uvm/assets/140069906/0f341a71-7844-46ee-a246-6d997a3531ea)

Can you please help me to resolve this error! Thank you

**Comment by MikeOpenHWGroup** [comment]
Hi @MFaayez, thanks for your interest in OpenHW and the CORE-V-MCU.  For a topic like this one, please create an _Issue_ on this GitHub repo.  We tend to use Issues for "things that do not work", and reserve Discussions for other topics such as feature requests, etc.

Also, in the Issue, please provide your complete set of command lines for reproducing the problem, starting with `git clone ....`.

Thanks!
